### PR TITLE
Organization launch priorities

### LINKED ISSUE #12432: Deleting an organization should add a warning
When deleting an organization, a warning should let the owner know to download invoices/receipts from Stripe. It should also indicate what will happen to the subscription plan - whether the plan will be cancelled automatically or if the user has to cancel the plan before deleting the organization.

### PR DESCRIPTION
**Pull request by ewdurbin**
Closes #12432 #12423 #12426 #12428

### CODE REVIEW

File: warehouse/templates/manage/organization/settings.html [hunk 1]
```diff
@@ -225,20 +225,21 @@ <h3>{% trans %}Cannot delete organization{% endtrans %}</h3>
...
-          You must transfer ownership or delete this project before you can delete your organization.
-        {% pluralize %}
-          You must transfer ownership or delete these projects before you can delete your organization.
-        {% endtrans %}
+        {% trans %}For each project, you must either:{% endtrans %}
+        <ul>
+          <li>{% trans %}Transfer ownership and remove the project from your organization{% endtrans %}</li>
+          <li>{% trans %}Delete the project{% endtrans %}</li>
+        </ul>
+        {% trans %}Before you can delete your organization.{% endtrans %}
```
Comment (miketheman) [inline warehouse/templates/manage/organization/settings.html:233]: **question:** ‏non-blocking, but is there a way to support translators in that this sentence is part of the one preceding the list items?

File: warehouse/organizations/models.py [hunk 2]
```diff
@@ -417,6 +417,9 @@ def active_subscription(self):
         else:
             return None
 
+    def customer_name(self, site_name="PyPI"):
+        (f"{site_name} Organization -" f"self.display_name ({self.name})"),
```
Comment (miketheman) [inline warehouse/organizations/models.py:421]: **suggestion:** ‏Correct f string representation here
```suggestion
        (f"{site_name} Organization - {self.display_name} ({self.name})"),
```

Curious why this wasn't caught by test - do we not assert on the outcome of the call of this function somewhere?

**Review by miketheman — APPROVED** [review]
One change requested with a fix. I'll approve so as to not block when corrected, since I'll be offline by then.

File: warehouse/organizations/models.py (hunk 2 above)
Comment (ewdurbin) [reply warehouse/organizations/models.py:421]: will add an assertion. good catch(es)

File: warehouse/templates/manage/organization/settings.html (hunk 1 above)
Comment (ewdurbin) [reply warehouse/templates/manage/organization/settings.html:233]: I pondered this as well, and tried

```
diff --git a/warehouse/locale/Makefile b/warehouse/locale/Makefile
index 844ea9f9e..6f511c2f3 100644
--- a/warehouse/locale/Makefile
+++ b/warehouse/locale/Makefile
@@ -5,6 +5,7 @@ compile-pot:
        PYTHONPATH=$(PWD) pybabel extract \
                -F babel.cfg \
                --omit-header \
+                --add-comments="i18n" \
                --output="warehouse/locale/messages.pot" \
                warehouse
 
diff --git a/warehouse/templates/manage/organization/settings.html b/warehouse/templates/manage/organization/settings.html
index d383b2c72..41d0e90f4 100644
--- a/warehouse/templates/manage/organization/settings.html
+++ b/warehouse/templates/manage/organization/settings.html
@@ -225,11 +225,13 @@
         {% pluralize %}
           Your organization currently owns {{ count }} projects.
         {% endtrans %}
+        {# i18n: Continued with "Before you can delete your organization." after a list of requirements #}
         {% trans %}For each project, you must either:{% endtrans %}
         <ul>
           <li>{% trans %}Transfer ownership and remove the project from your organization{% endtrans %}</li>
           <li>{% trans %}Delete the project{% endtrans %}</li>
         </ul>
+        {# i18n: Continuation of a sentence "For each project, you must either:" before a list of requirements #}
         {% trans %}Before you can delete your organization.{% endtrans %}
       </p>
       <ul class="no-bottom-margin">
```

But the comments weren't picked up.

**Pull request merged**